api/printful/printful.php

```diff
@@ -8,12 +8,24 @@ public function __construct($printful_api_key = PRINTFUL_API_KEY) {
     }
 
     protected function send_request($URI, $data = false) {
+        $endpoint = 'https://api.printful.com/' . $URI;
+        $isOrder =substr($URI, 0, 6) == 'orders';

```

Comment:
Let's do a case insensitive regex match here instead, something like `preg_match('/orders$/i, $URI)` or whatever the correct syntax is